packages/spotlight/src/spotlight.js

```diff
@@ -489,7 +495,10 @@ const Spotlight = (function () {
 				window.addEventListener('keyup', onKeyUp);
 				window.addEventListener('mouseover', onMouseOver);
 				window.addEventListener('mousemove', onMouseMove);
-				if (platform.webos) document.addEventListener('webOSMouse', handleWebOSMouseEvent);
+				if (platform.webos) {
+					document.addEventListener('webOSMouse', handleWebOSMouseEvent);
```

Comment:
This will need to attach to the top-level document for use with iframes.

Reply:
of course :facepalm: